make/fw.mk

```diff
@@ -0,0 +1,286 @@
+##############################################################################
+# Build global options
+# NOTE: Can be overridden externally.
+#
+
+# Compiler options here.
+ifeq ($(USE_OPT),)
+  USE_OPT = -O2 -ggdb -fomit-frame-pointer -falign-functions=16 -std=gnu99 -D_GNU_SOURCE
+  USE_OPT += -DBOARD_OTG_NOVBUSSENS $(build_args)
+  USE_OPT += -fsingle-precision-constant -Wdouble-promotion -specs=nosys.specs
+endif
+
+# C specific options here (added to USE_OPT).
+ifeq ($(USE_COPT),)
+  USE_COPT =
+endif
```

Comment:
Make variables can be overridden without `ifeq` noise, all variables defined on the command-line takes precedence over the ones defined in the Makefile.

Reply:
These were inherited from moving the original Makefile to `fw.mk`. I'm happy to get rid of them.